clarify issuer can be an AS too (#451)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

: Data that is presented to a specific Verifier, derived from one or more Verifiable Credentials that can be from the same or different Credential Issuers. It can be of any Credential Format.

Credential Issuer (or Issuer):

> :  An entity that issues Verifiable Credentials. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Authorization Server and/or a Resource Server (see [@!RFC6749]).

Reviewer: IMHO, specification currently provides a clear definition that Credential Issuer is, first and foremost, a Resource Server, and it took some time (drafts) to de-associate Issuer from AS (updated sequence diagrams, removed c_nonce from token endpoint etc).

The fact that a Credential Issuer could also act a an AS (protecting itself) is an implementation detail and as such, perhaps related remark to the implementation consideration section could me add (keeping the current definition as is).

Reviewer 2: ```suggestion
:  An entity that issues Verifiable Credentials. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Resource Server (see [@!RFC6749]). The Credential Issuer might also act as an Authorization Server.
```

> :  An entity that issues Verifiable Credentials. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Authorization Server and/or a Resource Server (see [@!RFC6749]).

Reviewer 3: ```suggestion
:  An entity that issues Verifiable Credentials. In the context of this specification, the Credential Issuer acts as an OAuth 2.0 Resource Server, and may also act as an OAuth Authorization Server (see [@!RFC6749]).
```
